packages/logos-docusaurus-theme/src/client/components/mdx/Grid/Grid.tsx

```diff
@@ -40,12 +46,33 @@ export const Grid: { Item: typeof GridItem } & React.FC<GridProps> = ({
     <GridRoot {...props} className={clsx(props.className, 'mdx-grid')}>
       <div className="mdx-grid__actions">
         {actions}
-        <div className="mdx-grid__scroll">
+        <div
+          className={clsx(
+            'mdx-grid__scroll',
+            spaceBetween && 'mdx-grid__scroll--space-between',
+          )}
+        >
           <IconButtonGroup size="small" color="primary">
-            <IconButton size="small" onClick={scroll.bind(null, -1)}>
+            <IconButton
+              className={clsx(
+                leftLabel?.length &&
+                  'mdx-grid__scroll__button mdx-grid__scroll__button--left',
+              )}
```

Comment:
Only modifier class names - `--left` in this case - should be set conditionally. 
Also, the names don't follow BEM convention, please use: 

- `mdx-grid__scroll-button` instead of `mdx-grid__scroll__button`
- `mdx-grid__scroll-button--left` instead of `grid__scroll__button--left`